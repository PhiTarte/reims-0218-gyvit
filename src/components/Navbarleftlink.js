// TODO create navbar

import React from "react"
import { Nav, NavItem, NavLink } from "reactstrap"

const Navbarleftlink = () => (
  <div>
    <Nav vertical>
      <NavItem>
        <NavLink href="#">Link</NavLink>
      </NavItem>
    </Nav>
    <hr />
  </div>
)

export default Navbarleftlink
