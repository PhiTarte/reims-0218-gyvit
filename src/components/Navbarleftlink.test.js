import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import Navbarleftlink from "./Navbarleftlink"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Navbarleftlink />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders Navbarleftlink html", () => {
  const tree = renderer.create(<Navbarleftlink />).toJSON()
  expect(tree).toMatchSnapshot()
})
