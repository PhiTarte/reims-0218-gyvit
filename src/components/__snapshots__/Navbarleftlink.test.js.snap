// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navbarleftlink html 1`] = `
<div>
  <ul
    className="nav flex-column"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="#"
        onClick={[Function]}
      >
        Link
      </a>
    </li>
  </ul>
  <hr />
</div>
`;
